Dockerfile

```diff
@@ -457,6 +473,9 @@ RUN /build/scripts/install_extensions versions > /.image_config; \
     echo "BUILDER_URL=\"${BUILDER_URL}\"" >> /.image_config; \
     echo "BUILD_DATE=\"$(date -Iseconds)\"" >> /.image_config
 
+#    echo "PGAI_VERSION=\"${PGAI_VERSION}\"" >> /.image_config; \
```

Comment:
Should uncomment and move this up a few lines, just after PGBACKREST or so?

Reply:
ah yes. thanks